ai.finto.fi/index.html

```diff
@@ -163,7 +163,7 @@ <h3>{{ $t('suggestions_label') }}</h3>
 
           <project-select v-model="selected_project" :projects="projects"></project-select>
           <limit-input v-model="limit"></limit-input><br>
-          <language-select v-model="language"></language-select>
+          <language-select v-model="language" :selected_project="selected_project"></language-select>
```

Comment:
Renaming `selected_project` to `selectedProject` did not affect the props name on line 439.